docs/docs-content/integrations/metallb.md

```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
 
 MetalLB is a load-balancer implementation for bare metal [Kubernetes](https://kubernetes.io/) clusters, using standard routing protocols. This integration is recommended for the on-prem cloud(s) and will help external service(s) get an IP address when the service type is set as LoadBalancer.
 
-## MetalLB Pack Working Details:
+MetalLB deploys a controller and a speaker. The speaker is deployed as a DaemonSet that runs on all nodes.
 
-* The address set in pack values goes into a configMap **`config`** in **`metallb-system`** namespace. This configMap is used by the MetalLB controller and speakers as volume mounts.
 
-* Any changes to the address will get updated in the configMap. Our users may confirm this with this command:
+## Versions Supported
 
-		kubectl describe cm config -n metallb-system.
+<Tabs queryString="versions">
 
-* However, the controller and speaker pods are already running with a previous copy of the configMap and these deployments are not aware of the new changes made to configMap. To ensure the address change are reflected, we need to restart the controller and speaker pods so that they will fetch the new configMap and start assigning new addresses correctly.
+<TabItem label="0.13.x" value="0.13.x">
 
-* Run the following commands, which will help restart the controller and speaker:
+## Prerequisites
+
+- A Kubernetes cluster running Kubernetes 1.13.0 or later that does not already have network load-balancing functionality.
+
+
+- A cluster network configuration that can co-exist with MetalLB.
+
+
+- Some IPv4 addresses for MetalLB to hand out.

```

Comment:
```suggestion
- Ensure sufficient IPv4 addresses for MetalLB are available.
```
The sentence is a bit too informal. @ritawatson we should ask Kevin if has a number we can recommend.